packages/tools/client-debugger/client-debugger-chrome-extension/src/messaging/Constants.ts

```diff
@@ -6,4 +6,4 @@
 /**
  * The message {@link ISourcedDevtoolsMessage.source} for all messages posted by the Devtools extension.
  */
-export const extensionMessageSource: string = "fluid-client-debugger-devtools";
+export const extensionMessageSource: string = "fluid-devtools-extension";
```

Comment:
When debugging, the old string was confusing. Hopefully this will more clearly differentiate the (currently) two message sources.